TodoList/src/app/screens/BookList/index.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+
+import books from '../../../constants/books';
+
+import BookListLayout from './layout';
+
+class BooksList extends Component {
+  static navigationOptions = { title: 'Book List' };
```

Comment:
You don't need this. The tab already has a title. Ask me if you need clarification about this.